code/game/objects/items/kitchen.dm

```diff
@@ -82,6 +102,23 @@
 /obj/item/knife/kitchen
 	name = "kitchen knife"
 	desc = "A general purpose Chef's Knife made by SpaceCook Incorporated. Guaranteed to stay sharp for years to come."
+	var/special_fluff_cooldown = 0
+
+/obj/item/knife/kitchen/attack_self(mob/user)
+	if(special_fluff_cooldown < world.time) //don't spam with it
+		special_fluff_cooldown = world.time + 10 SECONDS
+		do_special_fluff(user)
+	. = ..()
+
+/obj/item/knife/kitchen/proc/do_special_fluff(mob/user)
```

Comment:
```suggestion
/obj/item/knife/kitchen/do_special_fluff(mob/user)
```
I am surprised it doesn't object to this, you shouldn't have `/proc/` in there when extending a proc from the parent type.